lib/api/completeMultipartUpload.js

```diff
@@ -433,6 +434,11 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                     metaStoreParams.versioning = options.versioning;
                     metaStoreParams.isNull = options.isNull;
                     metaStoreParams.nullVersionId = options.nullVersionId;
+                    const replicationInfo = getReplicationInfo(objectKey,
```

Comment:
Since this new processing is not related to versioning, could we rename the `versioningPreprocessing` function to some more generic name?